authoring/tools-for-developers.md

```diff
@@ -64,8 +64,7 @@ repos:
 ```
 
 ```{note}
-See [the flake8 hooks explanation](see: https://flake8.pycqa.org/en/latest/user/using-hooks.html) 
-for more details
+See <a href="https://flake8.pycqa.org/en/latest/user/using-hooks.html">the flake8 hooks explanation</a> for more details.
```

Comment:
why are we hard-coding HTML here? In general we should try to use markdown syntax...

Reply:
oh sorry, actually I copied/pasted the code you sent in the review and as it didn't work I thought it was some conflict with `note` section. I just fixed the typo and it worked. thanks! as it was the first time I am working with jupyter-book ... I was a little bit confused ... thanks for the review !